templates/web/base/admin/body_translation_edit.html

```diff
@@ -0,0 +1,31 @@
+[% INCLUDE 'admin/header.html' title=tprintf(loc('Edit %s Translation for %s'), language_name, body.name ) -%]
+
+<form method="post" action="[% c.uri_for('body', body_id ) %]" enctype="application/x-www-form-urlencoded" accept-charset="utf-8" id="category_edit">
+    <input type="hidden" name="translation_id" value="[% translation.id| html %]" >
+    <input type="hidden" name="token" value="[% csrf_token %]" >
+    <input type="hidden" name="posted" value="edit_translation" >
+    <input type="hidden" name="language" value="[% translation.lang| html %]" >
+    <p>
+    <strong>Original</strong>: [% body.name | html %]
```

Comment:
I don't think this line is needed as the body name is given in the title just above.

Although I have just realised that if you add a translation in the same language you are viewing the admin site in, then the main body.name in "Edit body" will of course change to be that (which I guess doesn't matter, but is a bit confusing). Hmm.